### DISCUSSION TITLE
SRSRAN with E2 agent

### DISCUSSION THREAD

**Discussion by mdalgitsis** [srsran/srsRAN_4G#1142] [General]
Hello all,

I see research papers implementing a modified version of SRSRAN with an E2 agent to enable RT-RIC integration.
As I am having some issues to replicate those implementations, I am wondering if SRSRAN version 22.10 comes with an E2 agent included.

Thank you in advance.

**Comment by andrepuschmann** [comment]
Hi @mdalgitsis,

in srsRAN 4G 23.4 we will have an E2 agent included - not with 22.10 though. 

Hope that helps.

**Reply by mdalgitsis** [reply]
thank you very much for your instant reply!

Can I open a new discussion in this repo asking for help for a forked SNSRAN with E2 agent from another repo? That community doesn't seem to respond at all.
